test/src/639-specialfiles/main

```diff
@@ -1,6 +1,7 @@
 cvmfs_test_name="Handling special files"
 
 export CVMFS_TEST_UNIONFS="overlayfs"
+export CVMFS_IGNORE_SPECIAL_FILES="true"
```

Comment:
This would better be in /etc/cvmfs/repositories.d/$CVMFS_TEST_REPO/server.conf.  Like

    echo "PARAM=value" | sudo tee -a /etc/cvmfs/...

We should also add a test that confirms that cvmfs aborts on special files if this parameter is not set